src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/PlaneWaveMinkowski.hpp

```diff
@@ -0,0 +1,9 @@
+// Distributed under the MIT License.
```

Comment:
You don't use this header anywhere.  Is there a reason to have it rather than having people include the Minkowsi.hpp and PlaneWave.hpp headers themselves?  (Having the cpp for the explicit instantiations makes sense.  I'm only questioning the hpp here.)

Reply:
The answer is consolidation of includes, I guess. That and it saves people remembering which two includes need to be made when using this data. The same choice has been made for all analytic data instantiations in here here, e.g. [`PlaneWave3DKerrSchild`](https://github.com/sxs-collaboration/spectre/blob/develop/src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/PlaneWave3DKerrSchild.cpp), [`RegularSphericalWave3DKerrSchild`](https://github.com/sxs-collaboration/spectre/blob/develop/src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/RegularSphericalWave3DKerrSchild.cpp).

Reply:
But you're not using it, you're including all the pieces separately.  If you want this file to be the thing that is included when you want to use the class, you should include it when you use the class.

Honestly, if I were using `ScalarWaveGr<ScalarWave::Solutions::PlaneWave<3>, gr::Solutions::Minkowski<3>>`, I would find it easier to remember to include `ScalarWaveGr.hpp`, `PlaneWave.hpp`, and `Minkowski.hpp` than `PlaneWaveMinkowski.hpp`.  (As is done in the test.)

This file might be useful if it also defined an alias for this class so people didn't have to know exactly which analytic solutions are used internally.  If we had something like `using PlaneWaveMinkowski = ScalarWaveGr<...>;` so that it hid the classes being used, then I think hiding the includes would make sense.

Reply:
I think I agree with you (and so does the unit test) that if I wanted to use the class `ScalarWaveGr<A, B>` I would intuitively include `ScalarWaveGr`, `A`, and `B` instead of some arbitrarily named header `ScalarWaveAB`. Later on I might find that such a header exists and replace these includes with it, but its definitely not intuitive. I also don't like having an alias for the same reason, a user may just find the flexibility of `ScalarWaveGr` much more easier to remember.

In that spirit, I have removed `PlaneWaveMinkowski.hpp` here. Going one step ahead, I have 2 other similar headears in this directory to uphold the new convention. Ironically enough, no other code was impacted by these headers' removal, clearly showing how unused they were!